docs/docs/examples/github/index.mdx

```diff
@@ -17,15 +17,15 @@ The first step is to define the reactive (aka _observable_) state of your UI. Th
 In this case, we want a simple UI where we accept the _github-ID_ as input. We use that to fetch the user's repositories. This call will be made using the <PubBadge name="github" /> package, the results for which are stored as a `List<Repository>`. The `GithubStore` class so far looks like so:
 
 ```dart
```

Comment:
I think all the files from `docs` should not be part of this PR. Possibly because you checked out from the master branch first and then switched to null_safety ? Same goes for the README.md and other files not in `mobx_codegen`

Reply:
ok

Reply:
so I just delete all files except in mobx_codegen?

Reply:
done

Reply:
> so I just delete all files except in mobx_codegen?

No, don't delete, else it will be a deletion when merged onto master. I think you need to revert all changes outside of mobx_codegen or possibly do a fresh checkout from null-safety and then merge changes from the mobx_codegen folder ?